packages/playground-ui/src/ds/components/Composer/composer-ring.css

```
@@ -0,0 +1,79 @@
+.composer-ring:focus-within {
+  background-color: var(--border2);
+}
+
+.composer-ring:hover,
+.composer-ring:focus-within {
+  --composer-ring-strength: 0.4;
+}
+
+.composer-ring[data-busy='true'] {
+  --composer-ring-strength: 0.65;
+  animation: composer-ring-spin 2.4s linear infinite;
```

Stylelint (17.14.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)